cypress/e2e/awx/resources/workflowVisualizer.cy.ts

```diff
@@ -166,54 +141,274 @@ describe('Workflow Job templates visualizer', () => {
+              cy.get(`g[data-id=${projectNode.id}] .pf-topology__node__action-icon`).click({
+                force: true,
+              });
+              cy.get('li[data-cy="edit-node"]').click();
+              cy.selectDropdownOptionByResourceName('node-type', 'Inventory Source Sync');
+              cy.selectDropdownOptionByResourceName(
+                'inventory-source-select',
+                `${inventorySource.name}`
+              );
+              cy.selectDropdownOptionByResourceName('node-convergence', 'All');
+              cy.get('[data-cy="node-alias"]').type('Inventory Source Node');
+              cy.clickButton('Next');
+              cy.clickButton('Finish');
+              cy.get(`g[data-id=${projectNode.id}]`).should('have.text', 'Inventory Source Node');
+            });
+        }
+      );
+      cy.clickButton('Save');
+      cy.get('[data-cy="alert-toaster"]').should(
+        'have.text',
+        'Success alert:Successfully saved workflow visualizer'
+      );
+
+      cy.get('button[data-cy="workflow-visualizer-toolbar-close"]').click();
+      cy.get('[data-cy="page-title"]').should('have.text', `${workflowJobTemplate.name}`);
+    });
+  });
+  it('Create a job template node using a JT with multiple dependencies and then edit the node to use a different resource', function () {
+    cy.visit(`/templates/workflow_job_template/${workflowJobTemplate?.id}/visualizer`);
+    cy.contains('Workflow Visualizer').should('be.visible');
+    cy.clickButton('Add node');
+
+    cy.selectDropdownOptionByResourceName('node-type', 'Job Template');
+    cy.selectDropdownOptionByResourceName('job-template-select', `${jobTemplate.name}`);
+    cy.selectDropdownOptionByResourceName('node-convergence', 'All');
+    cy.get('[data-cy="node-alias"]').type('Test Node');
+    cy.clickButton('Next');
+    cy.clickButton('Finish');
+    cy.get('g[data-id="1-unsavedNode"]').should('have.text', 'Test Node');
+    cy.get(`g[data-id="1-unsavedNode"] .pf-topology__node__action-icon`).click({
+      force: true,
+    });
+    cy.get('li[data-cy="edit-node"]').click();
+    cy.selectDropdownOptionByResourceName('node-type', 'Project Sync');
+    cy.selectDropdownOptionByResourceName('project', `${project.name}`);
+    cy.selectDropdownOptionByResourceName('node-convergence', 'All');
+    cy.get('[data-cy="node-alias"]').type(`Project Node`);
+    cy.clickButton('Next');
+    cy.clickButton('Finish');
+    cy.get('g[data-id="1-unsavedNode"]').should('have.text', 'Test NodeProject Node');
+    cy.clickButton('Save');
+    cy.get('[data-cy="alert-toaster"]').should('be.visible');
+    cy.get('button[data-cy="workflow-visualizer-toolbar-close"]').click();
+
+    cy.get('[data-cy="page-title"]').should('have.text', `${workflowJobTemplate.name}`);
+  });
+  it('Adds a new node specifically linked to an already existing node.', function () {
+    let projectNode: WorkflowNode;
```

Comment:
There is a test on line 114 of this file: 'Adds a new node linked to an existing node with always status, and save the visualizer.' 
These two tests are very similar. Perhaps we can combine them and kill two birds with one stone? :-)